Extending example

eyaltrabelsi opened the pull request:
- improve(documentation): minimal requirements, creating table using spark dataframe, loading iceberg table using spark

rdblue commented:
This changes the notebook language to Python and duplicates a cell. I don't think it is something that's ready to merge. There's also a ton of extra metadata.

Also, there's now a published version, [0.2.0](https://jitpack.io/#Netflix/iceberg), in JitPack so there's no need to duplicate build instructions here.

eyaltrabelsi commented:
1. I removed the build instructions
2.  I restored the original notebook kernel metadata
3. I removed the redundant metadata I added
4. regarding the duplicates a cell do you mean these (Drop and create a table in HDFS using hive tables/spark dataframe)?  do you want me to write it a bit more generic ? to reserve imports only on one cell? remove the spark example ?

eyaltrabelsi commented:
@rdblue how does it look now ?
